plugins/node/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -150,8 +150,8 @@ export class MongoDBPlugin extends BasePlugin<typeof mongodb> {
       return MongodbCommandType.CREATE_INDEXES;
     } else if (command.findandmodify !== undefined) {
       return MongodbCommandType.FIND_AND_MODIFY;
-    } else if (command.ismaster !== undefined) {
-      return MongodbCommandType.IS_MASTER;
+    } else if (command.isprimary !== undefined) {
```

Comment:
The `isprimary` property doesn't exist on the command (at least in the driver's repo: https://github.com/mongodb/node-mongodb-native/search?q=isprimary&unscoped_q=isprimary), so i'm guessing renaming this will break the condition